A persona chapter in the Project design file (#1252)
alan-turing-institute/the-turing-way: book/website/project-design/persona-creation.md

- What hobbies and interests your persona has?
- What personality/demeanor (such as introvert, extrovert) matches your persona's?
- Through which communication channel will your persona potentially find out about the event?
  - Some options: by email, mailing list or forums, Twitter, Facebook, Slack, Gitter, newsletters, website or community repository
- What keeps your persona motivated on a typical day?

### Section 5: Resources and opportunities

This section will reflect on the resources available to your persona to participate in this project by asking the following questions:

- What resource can you offer your persona to participate in the project?
- Are there funding or skill-building opportunity that your persona can share more widely.
- What other resources does your persona need to fully participate with?

### Section 6: Hopes and dreams

This section will reflect on the expectations, goals, and emotions of your persona in regards to this project by asking the following questions:

- What does your persona want to achieve from this project?
- What information does your persona want to gain from their participation?

> - What fear or anxiety your persona may have regarding the project?

Reviewer: ```suggestion
- What fear or anxiety may your persona have regarding the project?
```

- What could inspire your persona at the project?
- How will your persona know that their goal has been achieved?

### Section 7: Perceived Barriers (negative trends)

This section will reflect on the perceived barriers that your persona may face in regards to this project by asking the following questions:

- What can frustrate your persona on a typical day?
- Why would your persona not join this project?
- What is speaking against this project that can discourage your persona from joining?
- What could alienate your persona from this project?
- What improvements can be introduced to help your persona overcome these barriers?

### Section 8: Hurdles in the project (headaches)

This section will reflect on the hurdles and issues that your persona may encounter in regards to this project by asking the following questions:

- What challenges does your persona face when participating in this project?
- What are your persona's hurdles specific to this project?
- How can your persona be supported in overcoming the challenges?
- How can we surpass your persona's expectations?

### Section 9: Fears and personal issues

> This section will reflect on fears and personal issues that your persona may face while participating in your project or community online by asking the following questions:

Reviewer: ```suggestion
This section will reflect on fears and personal issues that your persona may face while participating in this project or community online by asking the following questions:
```

- Would your persona be able to participate for a long period?
- In what situations your persona might encounter difficulties during their participation?
- What interruptions they might face while participating in this project?
- What are the resources available to support your persona better (like funding or equipment)?
- How can we assist your persona in reaching their goals?

## Putting this together in a persona canvas

> The persona canvas can be used to assemble all your response in one place, share this tangible information of your mental model (abstract concepts from our thoughts) with your colleagues and create a common language to communicate about your community members, users, and contributors.

Reviewer: ```suggestion
The persona canvas can be used to assemble all your responses in one place, share this tangible information of your mental model (abstract concepts from our thoughts) with your colleagues and create a common language to communicate about your community members, users, and contributors.
```

> The [https://www.designabetterbusiness.tools](https://www.designabetterbusiness.tools) have created multiple persona canvas (one is shown in the figure above) which are available for reuse under CC-BY ShareAlike 4.0 License, which can be downloaded [here](https://www.designabetterbusiness.tools/tools/persona-canvas).

Reviewer: ```suggestion
The [https://www.designabetterbusiness.tools](https://www.designabetterbusiness.tools) have created multiple persona canvases (one is shown in the figure above) which are available for reuse under CC-BY ShareAlike 4.0 License, which can be downloaded [here](https://www.designabetterbusiness.tools/tools/persona-canvas).
```

## Examples of personas in _The Turing Way_ community

> In the Community Handbook, we have provided a chapter on {ref}`Persona and Pathway <ch-persona>` for _The Turing Way_.

Reviewer: ```suggestion
In the Community Handbook, we have provided a chapter on {ref}`Personas and Pathways <ch-persona>` for _The Turing Way_.
```
